Changelog.md

```diff
@@ -1,5 +1,10 @@
 # Monoidal containers
 
+# 0.5.0.1
+
+  * Add a flag, `split-these`, to select whether to use the new "these"/"semialign" packages or the older combined "these" package.
```

Comment:
Note: this is automatic flag, i.e. `cabal` users don't need to care about setting it :)